src/components/card/card--flag.njk

```diff
@@ -0,0 +1,40 @@
+<ul class="usa-card-group">
+  <li class="tablet:grid-col-6 usa-card usa-card--flag">
+    <div class="usa-card__container">
+      <header class="usa-card__header">
+        <h2 class="usa-card__heading">Flag default</h2>
+      </header>
+      <div class="usa-card__media">
+        <div class="usa-card__img">
+          <img src="https://designsystem.digital.gov/img/introducing-uswds-2-0/built-to-grow--alt.jpg" alt="A placeholder image">
+        </div>
+      </div>
+
+      <div class="usa-card__body">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      </div>
+      <div class="usa-card__footer">
+        <button class="usa-button">Visit Florida Keys</button>
+      </div>
+    </div>
+  </li>
+  <li class="tablet:grid-col-6 usa-card usa-card--flag usa-card--media-right">
+    <div class="usa-card__container">
+      <header class="usa-card__header">
+        <h2 class="usa-card__heading">Flag media on right</h2>
+      </header>
+      <div class="usa-card__media">
+        <div class="usa-card__img">
+          <img src="https://designsystem.digital.gov/img/introducing-uswds-2-0/built-to-grow--alt.jpg" alt="A placeholder image">
+        </div>
+      </div>
+
+      <div class="usa-card__body">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      </div>
+      <div class="usa-card__footer">
+        <button class="usa-button">Visit Florida Keys</button>
+      </div>
+    </div>
+  </li>
+</ul>
```

Comment:
Do we control these files at all? Should we? If so, please add a newline at the end of this file.

Reply:
Reverting back to not changing these. But we pull most in automatically from USWDS